source/run-agg-pipelines.txt

```diff
@@ -140,69 +181,45 @@ First Stage
      The ``$project`` pipeline in the example suppresses the ``_id``
```

Comment:
It may be helpful to explicitly state how it suppresses the `_id` field. Maybe something like:

```
The ``$project`` stage specifies ``_id: 0`` to suppress the ``_id`` field
from the documents it passes to the next stage.
```

In this note, we can potentially add a seealso to link to https://docs.mongodb.com/manual/reference/operator/aggregation/project/#suppress-id-field-in-the-output-documents if you think it would be useful.

Reply:
updated